common/djangoapps/track/backends/tests/test_logger.py

```diff
@@ -47,6 +51,43 @@ def test_logger_backend(self):
         self.assertEqual(saved_events[0], unpacked_event)
         self.assertEqual(saved_events[1], unpacked_event)
 
+    @patch('track.utils.application_log')
+    def test_logger_backend_unicode_character(self, application_log):
+        """
+        When event information contain utf and latin1 characters
+        """
+        # Utf-8 characters
+        unicode_characters = {
+            'unicode': u'测试',
+            'string': '测试',
+            'encoded_unicode': u'测试'.encode('utf-8')
+        }
+        # Latin1 characters
+        latin_characters = {
+            'unicode_latin': u'Ó é ñ',
+            'string_latin': 'Ó é ñ',
+            'encoded_latin': u'Ó é ñ'.encode('latin1'),
+            'level_1': {'encoded_latin': u'Ó é ñ'.encode('latin1')}
+        }
+
+        self.backend.send(unicode_characters)
+        self.backend.send(latin_characters)
```

Comment:
Actually, instead of adding a test, just augment the one with latin_characters to be a dict that contains latin characters and unicode characters.  And a list of the same, now that I've discovered that many events also make use of lists.